C/manual/ch_Reports.xml

```diff
@@ -3778,7 +3778,179 @@ Income (type INCOME)
+            based on historical data, and the <emphasis>budgeted</emphasis> balance sheet is based
+            on the predictions made in the budget.
+          </para>
+        </abstract>
+
+        <para>The report contains the following elements:
+        </para>
+
+        <glosslist>
+          <title>Elements of Budgeted Balance Sheet</title>
+
+          <glossentry>
+            <glossterm>Assets - General</glossterm>
+            <glossdef>
+              <para>The value shown for each account is its balance as of the start date of the selected budget,
+                modified by the projected changes in this budget. So if all goes according to the
+                budget, these balances would be the same as if you were to run a standard Balance
+                Sheet as of the end date of the budget period.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Assets - Existing Assets</glossterm>
+            <glossdef>
+              <para>The sum of all budgeted asset accounts as of the start date of the budget. This is equivalent to the
+                Total Assets on the Balance Sheet when run for that start date.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Assets - Allocated assets</glossterm>
+            <glossdef>
+              <para>The sum of all Asset row totals projected in the budget.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Assets - Unallocated Assets</glossterm>
+            <glossdef>
+              <para>Of the profit you're projecting (New Retained Earnings as described below, which is subtracting
+                planned Expenses from planned Income), this field shows how much your (projected
+                Assets - projected Liabilities) are off by. If you're planning to net $100 monthly
+                ($1000 income and $900 expenses), and every hypothetical month your bank account
+                only goes up by $25 and you pay your credit card down by $26 (a combined increase of
+                only $51), you have a projected $49 of extra proceeds in the Income Statement that
+                doesn't have a corresponding increase in the Balance Sheet. If everything goes
+                exactly according to <emphasis>this</emphasis> budget, <emphasis>as-is</emphasis>,
+                either the Balance Sheet will grow by $49/month more than you're projecting (the
+                projected income/expenses are right), or the Income Statement for the whole period
+                will show a $51/month gain instead of $100 (the projected assets/liabilities are
+                right). This formula is:
+                <informalequation>
+                  <mathphrase>Unallocated Assets = New Retained Earnings - Allocated Assets - New
```

Comment:
Nitpick from <q>Hyphens and dashes</q> in [Text conventions](https://wiki.gnucash.org/wiki/Documentation_Improvement#Text_conventions):
`New Retained Earnings &minus; Allocated Assets &minus; New`
There multiple other occurrences.